src/include/ndpi_typedefs.h

```diff
@@ -958,6 +958,9 @@ struct ndpi_flow_udp_struct {
   /* NDPI_PROTOCOL_MUMBLE */
   u_int8_t mumble_stage:1;
   u_int64_t mumble_ident;
+
+  /* NDPI_PROTOCOL_LOLWILDRIFT */
+  u_int8_t lolwildrift_stage:1;
```

Comment:
Try moving it up, as `u_int32_t lolwildrift_stage:1` (note the 32, not 8...)  just before the epic games stuff.
This way the size of the structure shouldn't change

Reply:
> Try moving it up, as `u_int32_t lolwildrift_stage:1` (note the 32, not 8...) just before the epic games stuff. This way the size of the structure shouldn't change

Ty, seems that it works.